pytest_fixtures/component/activationkey.py

```diff
@@ -6,8 +6,14 @@
 
 
 @pytest.fixture(scope='module')
-def module_activation_key(module_org, module_target_sat):
-    return module_target_sat.api.ActivationKey(organization=module_org).create()
+def module_activation_key(module_entitlement_manifest_org, module_target_sat):
```

Comment:
```suggestion
def module_activation_key(module_sca_manifest_org, module_target_sat):
```

Reply:
I agree with your point @vsedmik that `6.16` onwards only SCA based orgs will be used and we need to use organization fixture accordingly which supports this feature. 
However this suggested changes is applicable in registration scenario of `6.12.z`, `6.13.z`, `6.14.z` and there may be customers who uses **SCA disbale org** & entitlement mode manifest while registration, so I wouldn't suggest to use of `SCA mode manifest org` and would ask to make separate change for later versions, 6.15.z and stream.
@shweta83

Reply:
@vijaysawant We have option to enable SCA in older versions, but in new newer versions, it will be enabled by default. There are many test cases where the manifest with SCA disabled are used so we are good from coverage pov.

Reply:
Use of `*_sca_manifest_org` over `*_entitlement_manifest_org` would have been consider by looking at `e2e` marker and label `6.12.z`, `6.13.z` & `6.14.z`
If we have another tests which are covering SCA-disable org scenario for `6.12.z`, `6.13.z` & `6.14.z` Satellite versions then we should be good with this PR change.

ACK